fxmanifest.lua

```diff
@@ -1,8 +1,9 @@
 fx_version 'cerulean'
 game 'gta5'
 
+author 'Original author unknown. Changes by Harry The Bastard'
```

Comment:
No

Reply:
What is the issue specifically, my name or the presence of the entry?

Reply:
I've made the assumption that the line should be completely removed.